### ISSUE TITLE
[BUG] Entities not showing

### ISSUE THREAD

**Issue by blairjanis** [marklogic-community/data-hub-central-community#218]
*### Welcome Data Hub Central Community Edition(DHCCE) enthusiast!

Let's get the most out of this ticket, please fill out this template.

#### The issue

Short description of the problem:

On the Integrate page I'm only seeing one entity under Manage My Data.  And that is the only entity available in the mapping step.  I expected to see all of the entities there.

What are the steps to reproduce the issue:
1) Create entities on the Connect page
2) Navigate to the Integrate page

#### Tech details

Which Operating System are you using? MacOS Catalina

Which version of MarkLogic are you using? 10.0.6

Which version of the Data Hub are you using? Data Hub Central 5.4.1/DHCCE v2.0.5

#### The devil is in the details...

If possible, rerun the command with -d for debugging output and attach the output:

  `java -jar dhcce.jar -d > myoutput.txt`

**Comment by wpaven** [comment]
Hi @blairjanis, 
You have to use DHCCE 5.4.1 with Data Hub Central 5.4.1.
v2.0.5 is compatible with Data Hub 5.2.x and not 5.4
When you spin up community edition, make sure the latest modules get deployed as well.

java -DforceInstall=true -jar hub-central-community-5.4.1.jar

Hope this helps,
Pete

**Comment by blairjanis** [comment]
So not a bug ... PICNIC (problem in chair, not in computer) :-)

There is some disconnect in the documentation and links in various recourses.  The link to Jar file on the main page of this repository downloads v2.0.5.  I see on here (https://marklogic-community.github.io/data-hub-central-community/) a comment about v5.4.1 of DHCCE but the only link takes you back to the repository with the link to the old file.  So I'm kind of going in circles trying to find the correct Jar file.

I found a link to this page (https://github.com/marklogic/marklogic-data-hub/releases) but these appear to be the Data Hub files, not the community file.

I finally found the 5.4.1 community jar file at https://github.com/marklogic-community/data-hub-central-community/releases.  I got there by copying the link on the README and stripping everything off of the link after "releases."  That link still points to /envision/ but the redirects ultimately got me to where I needed to be.
